src/NServiceBus.Metrics/RawData/OccurrenceWriter.cs

```diff
@@ -0,0 +1,42 @@
+namespace NServiceBus.Metrics.RawData
+{
+    using System;
+    using System.IO;
+
+    static class OccurrenceWriter
+    {
+        const long Version = 1;
+
+        // WIRE FORMAT, Version: 1
+
+        //0                   1                   2                   3
+        //0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
```

Comment:
Let me phrase my assumption in a following way: a `metric type` uses one of the `message formats`. This means that the receiver needs to know what is a `message format` for a specific `metric type`.

Answering your question. Yes, one metric means one message type. Is there something wrong with it?